exposee-app/server.js

```diff
@@ -134,27 +172,38 @@ app.post('/videos', validate_Token,async (req, res) => {
 
 app.post('/broadcast',validate_Token, async (req, res)=> {
   try {
-    const {url, user_Id, description, duration, api_key} = req.body;
+    const { user_Id, description, api_key} = req.body;
   
 
-    if (!url|| !description|| !duration || !user_Id||!api_key)
+    if ( !description || !user_Id||!api_key)
```

Comment:
nit user_Id should just be user_id

is there any way to get the user ID from the JWT token/user, rather than passing it in?

Reply:
thank was why i added the user_id to the payload